packages/playwright-core/src/cli/program.ts

```diff
@@ -159,6 +159,8 @@ program
         } else {
           const forceReinstall = hasNoArguments ? false : !!options.force;
           await registry.install(executables, forceReinstall);
+          for (const executable of executables)
+            await executable.validateHostRequirements(process.env.PW_LANG_NAME || 'javascript');
```

Comment:
- This now prints `Skipping host requirements validation logic because...` multiple times.
- `Cannot launch Firefox on Ubuntu 16.04! Minimum required Ubuntu version for Firefox browser is 20.04` error is now misplaced. We should leave it for the launch time.
- Which brings a question whether we'll get some false positives for unused WebKit installations.

Reply:
1 addressed, 2 removed, since we don't do things like this for other browsers, and 16.04 is really outdated. For 3, I improved the error message like that we can clarify it with others, what they think:

```
root@d8d476073591:/work# npx playwright install
Failed to install browsers
Error: 
╔════════════════════════════════════════════════════════╗
║ Host system is missing dependencies to run browsers.   ║
║ Please install them with the following command:        ║
║                                                        ║
║     npx playwright install-deps                        ║
║                                                        ║
║ Alternatively, use apt:                                ║
║     apt-get install libnss3\                           ║
║         libnspr4\                                      ║
║         libdbus-1-3\                                   ║
║         libatk1.0-0\                                   ║
║         libatk-bridge2.0-0\                            ║
║         libcups2\                                      ║
║         libdrm2\                                       ║
║         libxkbcommon0\                                 ║
║         libatspi2.0-0\                                 ║
║         libxcomposite1\                                ║
║         libxdamage1\                                   ║
║         libxfixes3\                                    ║
║         libxrandr2\                                    ║
║         libgbm1\                                       ║
║         libasound2                                     ║
║                                                        ║
║ Or just install the browsers you actually want to run: ║
║                                                        ║
║     npx playwright install chromium                    ║
║                                                        ║
║ <3 Playwright Team                                     ║
╚════════════════════════════════════════════════════════╝
```